package.json

```diff
@@ -67,11 +93,22 @@
   },
   "engineStrict": true,
   "prettier": {
-    "arrowParens": "always",
     "printWidth": 100,
     "proseWrap": "always",
     "singleQuote": true
   },
+  "eslintConfig": {
+    "extends": [
+      "airbnb-typescript",
+      "airbnb/hooks",
+      "prettier",
+      "prettier/@typescript-eslint",
+      "prettier/react"
```

Comment:
I don't have much experience using `eslint` in a TypeScript project. Most of my experience being in disagreement with one opinion or another.

If we introduce `eslint`, I would prefer to do so in a very minimal way. I feel like `tsc` already does a good job of catching those things which are _actually_ broken. I've also found that the 'hooks' linting from CRA was helpful.

I would prefer to strip this back to the bare minimum or even defer `eslint` altogether.